CHANGES.rst

```diff
@@ -2,6 +2,14 @@
 Changelog
 =========
 
+v0.8.1 (unreleased)
```

Comment:
```suggestion
v0.9.0 (unreleased)
```
Or do you think that we should publish this ASAP?

Reply:
Absolutely no need to publish now. I just wanted to update these things while I was looking at them.